README.md

```diff
@@ -162,7 +162,7 @@ This configuration includes:
   - **git**: Installs Git to manage source code versioning.
 - **overrideFeatureInstallOrder**: Specifies the order of feature installation to ensure common utilities, Git, Node.js, and TypeScript are set up in the correct sequence.
 - **customizations**: Installs essential Visual Studio Code extensions, including Prettier, ESLint, a spell checker, and TypeScript Next.
-- **workspaceFolder**: Sets the workspace folder to `/workspaces/starter-backstage` to match the local workspace folder name.
+- **workspaceFolder**: Sets the workspace folder to `/workspaces/${localWorkspaceFolderBasename}` to match the local workspace folder name.
```

Comment:
`"workspaceFolder": "/workspaces/daytona-backstage-starter"` this is set in the `devcontainer.json` file

Reply:
Thankyou for pointing that out. I have updated both `devcontainer.json` and `readme.md` for this branch.